clusters/veritable-prod/alice/ui/release.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ui
+  namespace: alice
+spec:
+  install:
+    remediation:
+      retries: -1
+  releaseName: ui
+  chart:
+    spec:
+      version: 1.0.5
+      chart: veritable-ui
+      sourceRef:
+        kind: HelmRepository
+        name: digicatapult
+  interval: 10m0s
+  # Default values
+  # https://github.com/bitnami/charts/blob/master/bitnami/nginx-ingress-controller/values.yaml
+  valuesFrom:
+    - kind: ConfigMap
+      name: alice-values
+      valuesKey: ui-values.yaml
+    - kind: Secret
+      name: alice-companies-house-api
+      valuesKey: key
+      targetPath: companysHouseApiKey.secret
```

Comment:
what does this mean that the secret is on  some path `companysHouseApiKey.secret` ? as in that is where the app can find it or sth like that?

Reply:
`- kind: Secret` What we are going to use to provide a value, either a configMap or a Secret
`name: alice-companies-house-api` the name of the secret to lookup to provide the value
`valuesKey: key` in the secret there are a set of data objects, this is the key to lookup
`targetPath: companysHouseApiKey.secret` the destination key in the helm chart to insert this value to.